src/View/Requirements_Backend.php

```diff
@@ -398,6 +401,16 @@ public function setMinifyCombinedFiles($minify)
      */
     public function javascript($file, $options = array())
     {
+        // String of the form vendor/package:resource. Excludes "http://bla" as that's an absolute URL
```

Comment:
We should put this and the below block into a single method to reduce duplication.